init_test.go

```diff
@@ -10,7 +10,7 @@ import (
 func TestUnit(t *testing.T) {
 	suite := spec.New("poetry", spec.Report(report.Terminal{}), spec.Parallel())
 	suite("Build", testBuild)
-	suite("Detect", testDetect)
+	suite("Detect", testDetect, spec.Sequential())
```

Comment:
Do we have to run the tests in order? I'm surprised to see the `spec.Sequential()` in unit tests.

Reply:
The original test called out `spec.Sequential()`. I tried removing it but the tests failed, I suspect because the test is manipulating the environment variables.

Reply:
Ok, we can keep this for now and investigate after we merge.

Reply:
Can confirm this is an issue we've seen across buildpacks and as far as we can tell does pertain to the manipulation of environment variables.

Reply:
Yes, I realized that after I wrote my comment.